pkg/resources/rbac.authorization.k8s.io/v1/clusterrolebinding/validator_test.go

```diff
@@ -0,0 +1,173 @@
+package clusterrolebinding
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/rancher/webhook/pkg/admission"
+	"github.com/stretchr/testify/require"
+	admissionv1 "k8s.io/api/admission/v1"
+	v1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	gvk = metav1.GroupVersionKind{Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "RoleBinding"}
+	gvr = metav1.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "rolebindings"}
+)
+
+func TestAdmit(t *testing.T) {
+	t.Parallel()
+
+	defaultRoleBinding := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+		},
+	}
+	emptyLabelsRoleBinding := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{},
+			Name:   "default",
+		},
+	}
+	roleBindingWithOwnerLabel := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+			Labels: map[string]string{
+				grbOwnerLabel: "grb-owner",
+			},
+		},
+	}
+	roleBindingWithNewOwnerLabel := &v1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+			Labels: map[string]string{
+				grbOwnerLabel: "new-owner",
+			},
+		},
+	}
+
+	type args struct {
+		oldRB *v1.ClusterRoleBinding
+		newRB *v1.ClusterRoleBinding
+	}
+	type test struct {
+		name    string
+		args    args
+		allowed bool
+		wantErr bool
+	}
+	tests := []test{
```

Comment:
Similar comment about handling the error case.

Reply:
added